crates/node_binding/binding.d.ts

```diff
@@ -350,6 +350,7 @@ export interface JsStatsModule {
   issuerPath: Array<JsStatsModuleIssuer>
   reasons?: Array<JsStatsModuleReason>
   assets?: Array<string>
+  source?: string
```

Comment:
Why is it always a `string` typed? It should be `Buffer` when it comes to binary assets.

Reply:
> I don't know what to do now. Can you help me? There is a `JsCompatSource` object in `node_binding`, which I want to use, but in the `stats` module of`rspack_core`, `StatsModule` can only get an `Option<&dyn Source>` This `Source` In this case, if I want to implement `&dyn Source` I have to add a reference to StatsModule, I don't think this solution is very good, do you have any suggestions?

Sure, I will take a look in these days.